### DISCUSSION TITLE
When to use uBlock Origin and when to use uBO in the wiki?

### DISCUSSION THREAD

**Discussion by androidkotlindev** [uBlockOrigin/uBlock-issues#1966] [Q&A] [answered]
I wonder how uBlock Origin and uBO should be used in wiki?
Should they be used like "uBlock Origin is used in the first sentence of a paragraph and uBO is used in other sentences"?

**Comment by ghost** [comment]
> "uBlock Origin is used in the first sentence of a paragraph and uBO is used in other sentences".

More or less? Once under linkable headline? Can be skipped in advanced topics?

**Comment by uBlock-user** [comment]
Doesn't matter, both are interchangeable.

**Comment by gorhill** [comment, marked as answer]
In general I use "uBlock Origin" -- even better "uBlock Origin (uBO)" -- in first instance in the text, then I use uBO afterward. I want "uBO" to become widely used to refer to uBlock Origin, so if ever Eyeo decides to enforce their [trademark ownership of "uBlock"](https://euipo.europa.eu/eSearch/#details/trademarks/017822487), uBO in widespread use can be used by then.

**Reply by androidkotlindev** [reply]
Should Adblock Plus/ABP be used similarly on the wiki pages?

**Reply by gorhill** [reply]
Yes, it's common practice to use full name with abbreviation in parenthesis in first instance, then only abbreviation afterward.
